src/components/Agencies/Agencies.tsx

```diff
@@ -1,3 +1,13 @@
+import { Container, Grid, Paper } from '@material-ui/core';
 import React from 'react';
 
-export const Agencies = () => <div>Agencies</div>;
+export const Agencies = () => (
+  <>
+    <h1>Agencies</h1>
+    <Paper>
+      <Container>
+        <Grid container></Grid>
+      </Container>
+    </Paper>
+  </>
+);
```

Comment:
this components isn't used

Reply:
she fixed that in the next commit

Reply:
try to sign commits more informatively

Reply:
@vovkis13 it was done in next commit
@topsart commit was renamed